apps/ledger-live-mobile/src/components/RootNavigator/AddAccountsNavigator.tsx

```diff
@@ -29,14 +29,14 @@ const totalSteps = "3";
 export default function AddAccountsNavigator({ route }: { route: Route }) {
```

Comment:
is it possible to use the opportunity to properly type the navigation to this route ? (I think @LFBarreto is working on this so maybe ask him if he's not already handling this Navigator)

cf. official doc https://reactnavigation.org/docs/typescript/